users/migrations/0001_initial.py

```diff
@@ -18,7 +18,7 @@ class Migration(migrations.Migration):
                 ('id', models.BigAutoField(auto_created=True,
                  primary_key=True, serialize=False, verbose_name='ID')),
                 ('name', models.CharField(max_length=255,
-                 blank=False, default=None)),
+                 blank=False, default=None, unique=True)),
```

Comment:
Removing this part breaks the pytest pipeline - `test_create_team_without_title` fails in this case.
In this test we try to create a team without passing a value to 'name' field. I tried to figure out what is the default value for a CharField (without explicitly define `default=...`) in Django framework but couldn't find the answer online. After a few tries and errors, I think the default is "". 
Thus, I see two solutions here but am not sure which one is preferred:
1. Leave the `default=None` as it is. I can't see any drawbacks here, because `Team.objects.create` raises an error when passing None to 'name' field. That way we can be sure that there won't be any duplications with `name = None`.
2. Remove `default=None` and create a new class method to the Team model `create_team`, something like:
```
def create_team(cls, name, description):
    if (name == ""):
        raise Exception
    Team.objects.create(name=name, description=description)
```

Reply:
@orzionpour The second option makes sense to me because we limited this field with:
blank=False, 
unique=True